### PR TITLE
Explaining the two major ways to register an event

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Second attempt of https://github.com/doctrine/orm/pull/9097

### CODE REVIEW

File: docs/en/reference/events.rst [hunk 1]
```diff
@@ -121,6 +121,16 @@ Now you can test the ``$eventSubscriber`` instance to see if the
...
 
+Registering Events
+~~~~~~~~~~~~~~~~~~
+
+There are two ways to register an event:
+
+* *All events* can be registered by calling ``$eventManager->addEventListener()``
+or ``eventManager->addEventSubscriber()`` (see :ref:`Listening and subscribing to Lifecycle Events`)
+* *Lifecycle Callbacks* can also be registered by adding an annotation to the entity
+(see :ref:`Lifecycle Callbacks`)
```
Comment (SenseException) [inline docs/en/reference/events.rst]: You need to set the references atop the specific headlines like this:

```
.. _transitive-persistence:

Transitive persistence / Cascade Operations
-------------------------------------------
```

to be able to use something like

```
:ref:`transitive-persistence`
```

**Review by SenseException — CHANGES_REQUESTED** [review]

File: docs/en/reference/events.rst (hunk 1 above)
Comment (ThomasLandauer) [reply docs/en/reference/events.rst]: Thanks!

**Review by SenseException — APPROVED** [review]

**Comment by derrabus** [comment]
@ThomasLandauer can you rebase on 2.10.x and resolve the conflict?

**Comment by ThomasLandauer** [comment]
Closing in favor of https://github.com/doctrine/orm/pull/9128

**Pull request closed without merging**